apps/zotonic_mod_base/src/controllers/controller_id.erl

```diff
@@ -109,9 +111,42 @@ get_id(Context) ->
                 <<>> ->
                     {undefined, Context};
                 Id ->
-                    RscId = m_rsc:rid(Id, Context),
-                    {RscId, z_context:set(id, {ok, RscId}, Context)}
+                    case maybe_split_extension(Id) of
+                        {Root, Ext} ->
+                            RscId = m_rsc:rid(Root, Context),
+                            Context1 = z_cowmachine_middleware:set_accept_context(Ext, Context),
```

Comment:
The mapping of extension to mime-type may be one to many, e.g. .json for ld+json, but also for other json based serializations. Can we require a one to one mapping for disambiguation? What would be the common extension for ld+json? .ld.json?

Reply:
I have no idea what the extension for ld+json would be.  But a double extension wouldn't be ok.  So more like `.jsonld`

Reply:
My goodness https://www.file-extension.info/format/jsonld

Reply:
Aha it is `.jsonld` see https://json-ld.github.io/json-ld.org/spec/latest/json-ld/#iana-considerations

Reply:
Added the mapping to the media identify routines.